mlir/utils/buildbot/mlir-rocm-mi200/token

```diff
@@ -0,0 +1 @@
+rocmmlir22
```

Comment:
Password for the worker account.  Will be sent by separate email to the buildbot-master administrator when we're ready to start.